docs/cucumber.md

```diff
@@ -1,36 +1,64 @@
 #Cucumber:
  
- The first question that would rise for a beginner would be 
+The first questions for a beginner are usually something like:
  
   *1. What is Cucumber?*
   *2. How Cucumber works with BDD?*
```

Comment:
I have one change request; the question posed here isn't the same as the (better) heading you edited below.

Proposal: Change the question 2 to read "How does Cucumber work with BDD?"